### PR TITLE
Adding a scoped less-import for global header team

### LINKED ISSUE #55: Create scoped CSS
We need to provide distributables of CSS that are scoped to a supplied selector name, e.g.:

```css
.gh .menu {
  ..
}
.gh .menu__items {
  ..
}
```

These distributables can be opted into using a Lasso flag.

### PR DESCRIPTION
**Pull request by seangates**
## Description

Because the global header must remain "conflict free" with other apps we need to provide a scoped version of the DS6 components. This commit adds a `gh.browser.json` which will use `less-import` a "wrapped" version of the DS6 LESS.

This also moves the `/src` folder out of `.npmignore`, since it needs to be published in order for this to work.

## Context

This came from a request and discussions from the Global Header team.

## References

Closes #55

### CODE REVIEW

**Comment by ianmcburnie** [comment]
I just thought... instead of moving all the ds6 source files over into the dist folder, couldn't we just remove `src` from `.npmignore` and reference the src files where they are directly? Maybe I'm overlooking something though...

File: dist/gh.css [hunk 1]
```diff
@@ -0,0 +1,835 @@
+.ds6 {
```
Comment (ianmcburnie) [inline dist/gh.css]: This file doesn't seem to be referenced by any browser.json. How does the global header consume it?

**Comment by ianmcburnie** [comment]
I'm a little confused, in that we appear to providing both a less-import, and a gh.css. Which one does global header actually need?

File: dist/gh.css (hunk 1 above)
Comment (seangates) [reply dist/gh.css]: Correct. It was an unintended generated file by the Gulp scripts. I could delete it manually at the end, or write some gulp script to delete it. Either way, I didn't explicitly try to have Gulp create it.

**Comment by CestDiego** [comment]
Will GH be able to decide to include minimal sets of this new scoped bundle or will it have to include all of it? 

For example. Right now there is no need to use `dropdown` So it would be good to not send that down as well. Does this allow that?

**Comment by seangates** [comment]
@CestDiego Let's tailor this to what you need, as the `gh.browser.json` will be for the GH specifically.

**Comment by seangates** [comment]
I guess we could also create additional `*.browser.json` files for the Less imports individually, per component, but that's a bit of admin overhead I think we want to avoid (at the moment).

**Comment by seangates** [comment]
@CestDiego We've discussed the way this works, and I am going to make some changes so that you can pull them in individually.

**Comment by seangates** [comment]
On it.

**Comment by seangates** [comment]
@ianmcburnie Fixed.

**Comment by seangates** [comment]
@DylanPiercey Any objections here?

**Pull request merged**